src/MeshKernelNET/Api/IMeshKernelApi.cs

```diff
@@ -1075,14 +1075,23 @@ int Mesh2dGetClosestNode(int meshKernelId, double xCoordinateIn, double yCoordin
 
 
         /// <summary>
-        /// Gets the grid state as a <see cref="Mesh2DNative"/> structure including the cell information
+        /// Gets the grid state as a <see cref="DisposableMesh2D"/> structure including the cell information

```

Comment:
Document that vertex coordinates, edge connectivity, face connectivity, edge midpoints and face center points are initialized.